### PR TITLE
Added api call for wishlist and cart

### PR DESCRIPTION
**Pull request by anupkgurung**
Added APi call for 
Cart and wishlist  if user is logged in.

### CODE REVIEW

**Review by Rohitprasad83 — APPROVED** [review]
Code looks clean and readable. Good to merge

File: src/api/CartApi.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                operation: "ADD_TO_CART",
+                payLoad: product
+            })
+        }
+        else {
+            dispatchCart({
+                operation: "ADD_TO_CART",
+                payLoad: product
+            })
+        }
```
Comment (rohanmathur91) [inline src/api/CartApi.js:33]: Can you explain this

**Review by rohanmathur91 — APPROVED** [review]
Added one comment
- Code structure looks good
- Try to add toast/alerts instead of the console log 
- Overall looks good

File: src/api/CartApi.js (hunk 1 above)
Comment (anupkgurung) [reply src/api/CartApi.js:33]: it's just if user is not logged it than the same method will do the carddispatch. it's still is incomplete once the user goes to cart there will be further functionality that would ask user to login or could checkout as a guest and the cart value will be pushed to api.

**Pull request merged**